src/app/pages/auth/login/login.component.ts

```
@@ -20,8 +20,7 @@ export class LoginComponent {
   login() {
     this.authService.login(this.credentials).subscribe(
       () => this.router.navigate(['/admin/dashboard']),
-      (error) => this.errorMessage = 'Invalid credentials'
+      (error) => (this.errorMessage = 'Invalid credentials')
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)